Makefile

```diff
@@ -614,13 +614,13 @@ scalaclean:
 	(cd $(ROOTDIR)/scala-package && mvn clean)
 
 scalapkg:
-	(cd $(ROOTDIR)/scala-package && mvn install -DskipTests)
+	(cd $(ROOTDIR)/scala-package && mvn install -q -DskipTests)
```

Comment:
Still, adding -q would made developer hard to debug the code if there is any failures in middle. This PR is also stale since all CIs has changed from using Makefile to use `mvn` command directly. As Jenkins didn't have a restriction on the lines of logs to print, we should have as much as possible